tests/Hydra/Serializer/EntrypointNormalizerTest.php

```diff
@@ -54,9 +56,10 @@ public function testNormalize()
         $collection = new ResourceNameCollection([FooDummy::class, Dummy::class]);
         $entrypoint = new Entrypoint($collection);
 
-        $factoryProphecy = $this->prophesize(ResourceMetadataFactoryInterface::class);
-        $factoryProphecy->create(Dummy::class)->willReturn(new ResourceMetadata('Dummy', null, null, null, ['get']))->shouldBeCalled();
-        $factoryProphecy->create(FooDummy::class)->willReturn(new ResourceMetadata('FooDummy', null, null, null, ['get']))->shouldBeCalled();
+        $factoryProphecy = $this->prophesize(ResourceCollectionMetadataFactoryInterface::class);
+        // TODO: Should shortName be loaded automatically by a decorator or I am allowed to manually define it below ?
```

Comment:
define it manually is fine, I'd have prefered not to change this test but rather to add a new test using REsourceCollection

Reply:
J'ai remis l'ancien test tel qu'il était + rajouté un nouveau test contenant ce que j'ai écrit